docs/99-references/00-references.adoc

```diff
@@ -12,27 +12,43 @@ Dieser Abschnitt enthält Quellenangaben, die ganz oder teilweise im Curriculum
 This section contains references that are cited in the curriculum.
 // end::EN[]
 
-- [[[bachmann,Bachmann 2000]]] Bachmann, F., L. Bass, et al.: Software Architecture Documentation in Practice. Software Engineering Institute, CMU/SEI-2000-SR-004.
+**B**
 
-- [[[bass,Bass 2021]]] Bass, L., Clements, P. und Kazman, R.: Software Architecture in Practice. 4. Edition, Addison-Wesley 2021.
+- [[[bachmann,Bachmann 2000]]] F. Bachmann, L. Bass et al.: Software Architecture Documentation in Practice. Software Engineering Institute, CMU/SEI-2000-SR-004.
 
-- [[[brown-c4,Brown (C4)]]] Brown, S.:, R.: The C4 model for visualising software architecture. Leanpub <https://leanpub.com/visualising-software-architecture>
+- [[[bass,Bass 2021]]] L. Bass, P. Clements und R. Kazman: Software Architecture in Practice. 4. Edition, Addison-Wesley 2021.
 
-- [[[brown-sg,Brown (Guidebook)]]] Brown, S.:, R.: The software guidebook. A guide to documenting your software architecture. Leanpub <https://leanpub.com/documenting-software-architecture>
+- [[[brown-c4,Brown (C4)]]] S. Brown: The C4 model for visualising software architecture. Leanpub <https://leanpub.com/visualising-software-architecture>
```

Comment:
Wieso ist die URL eigentlich in spitzen Klammern?

Reply:
Das hatte ich mir bei bestehenden Einträgen hier in der Datei abgeguckt. Never change a running format. Ich mache die mal weg.